### PR TITLE
[Look and Feel] Mini banner on eMIB welcome and howto pages

### PR DESCRIPTION
**Pull request by MichaelCherry**
# Description

Adding the mini banner to eMIB welcome and howto pages, as per look and feel. Welcome page will likely be removed in a later PR, but adding it for consistency.

Also using H1 in place of H2.

## Type of change

Please delete options that are not relevant.

- Code cleanliness or refactor

## Screenshot

Before:

After:
image

# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Go to Prototype Page
2. Start eMIB Sample Test
3. Verify that banner is present on Welcome and HowTo page, above header.

Screenshot of unit tests run passing:
image

# Checklist

Applicable for all code changes.

- [x] My code follows the style guidelines of this project
- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [ ] ~~I have made corresponding changes to the documentation~~
- [ ] My changes generate no new compiler warnings
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [x] New and existing unit tests pass locally with my changes
- [x] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [x] My changes look good on IE 10+, Firefox, and Chrome

### CODE REVIEW

**Comment by fnormand01** [comment]
@MichaelCherry  Do we want this banner only displayed in eMIB sample test Home page and How To page? Or in the whole test?

**Comment by MichaelCherry** [comment]
@fnormand01 only on the instructions page. According to the wireframes, it is not visible in the actual test. Joey also confirmed this on slack

File: frontend/src/components/eMIB/Emib.jsx [hunk 1]
```diff
@@ -35,7 +36,11 @@ class Emib extends Component {
   render() {
     return (
       <div className="app">
-        <h2>{LOCALIZE.emibTest.homePage.testTitle}</h2>
+        {this.state.curPage === PAGES.welcome && (
```
Comment (caleybrock) [inline frontend/src/components/eMIB/Emib.jsx]: you could write this like:
```
{((this.state.curPage === PAGES.welcome) ||  (this.state.curPage === PAGES.howTo)) &&
  <img src={mini_banner} alt="" className="banner" />
)}
```

**Comment by fnormand01** [comment]
As discussed offline, according to the wireframes, the header is a lot bigger and this mini banner is closer to  the header.

However, we'll fix that in another Pull Request.

**Comment by fnormand01** [comment]
Once @caleybrock's comment will be answered, I'll be good with the changes.

File: frontend/src/components/eMIB/Emib.jsx (hunk 1 above)
Comment (MichaelCherry) [reply frontend/src/components/eMIB/Emib.jsx]: fixed in 4007e42

**Comment by MichaelCherry** [comment]
Fixed, merging so Francis and I can continue with the next steps

**Pull request merged**